docs/apis/http/oidc/_index.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "OIDC"
+date: 2023-10-10T00:00:00+00:00
+weight: 21
+geekdocRepo: https://github.com/owncloud/ocis
+geekdocEditPath: edit/master/docs/apis/http/idp
+geekdocFilePath: _index.md
+geekdocCollapseSection: true
+---
+
+Infinite Scale can act as an identity provider (IdP) for OpenID Connect (OIDC) authentication .
```

Comment:
ocis can make use of other IDPs but can it be considered as an IDP in itself? Or the sentence could be
```suggestion
Infinite Scale has implemented OpenID Connect (OIDC) for authentication.
```

Reply:
This suggestion makes it neutral, independent if internal or external. 👍